### DISCUSSION TITLE
Fillings with double ixbrl

### DISCUSSION THREAD

**Discussion by mrx23dot** [manusimidt/py-xbrl#67] [Q&A]
How to proceed with fillings with double ixbrl in them?
The main htm doesn't contain many numbers:

https://www.sec.gov/Archives/edgar/data/1051512/000105151221000009/0001051512-21-000009-index.htm

Should one always choose the last one if there is more than one?

**Comment by manusimidt** [comment]
I am not quite sure why in this particular submission the tagged financial statements are in the exhibit.
Normally the 10-K or 10-Q document should contain all financial statements, not the exhibit.

The 10-K document contains 73 facts and is successfully parsed by the libary.
When parsing the exhibit EX-13 the parser crashes for some reason. 
I have to check if the document follows the iXBRL Specification.

**Comment by mrx23dot** [comment]
related issue https://github.com/manusimidt/py-xbrl/issues/64
